### PR TITLE
Loki Release: update release notes and docs

### PR DESCRIPTION
**Pull request by slim-bean**

### CODE REVIEW

File: docs/sources/operations/storage/_index.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: Storage
 ---
 # Loki Storage
 
+[High level storage over view here]({{< relref "../../storage/_index.md" >}})
```
Comment (cyriltovena) [inline docs/sources/operations/storage/_index.md:6]:
```suggestion
[High level storage overview here]({{< relref "../../storage/_index.md" >}})
```

File: docs/sources/operations/storage/filesystem.md [hunk 2]
```diff
@@ -42,103 +42,3 @@ The durability of the objects is at the mercy of the filesystem itself where oth
 ### High Availability
 
 Running Loki clustered is not possible with the filesystem store unless the filesystem is shared in some fashion (NFS for example).  However using shared filesystems is likely going to be a bad experience with Loki just as it is for almost every other application.
-
```
Comment (cyriltovena) [inline docs/sources/operations/storage/filesystem.md:45]: Is that a not working anymore ? just curious.

File: docs/sources/upgrading/_index.md [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+# Upgrading Loki
+
+Every attempt is made to keep Loki backwards compatible, such that upgrades should be low risk and low friction.
+
+Unfortunately Loki is software and software is hard and sometimes we are forced to make decisions between ease of use and ease of maintenance.
+
+If we have any expectation of difficulty upgrading we will document it here.
+
+As more versions are released it becomes more likely unexpected problems arise moving between multiple versions at once. 
+If possible try to stay current and do sequential updates, if you want to skip versions maybe don't yolo it in prod.
```
Comment (cyriltovena) [inline docs/sources/upgrading/_index.md:15]:
```suggestion
If possible try to stay current and do sequential updates, if you want to skip versions maybe try it a development environment before attempting to upgrade production.
```

File: docs/sources/operations/storage/filesystem.md (hunk 2 above)
Comment (slim-bean) [reply docs/sources/operations/storage/filesystem.md:45]: I'm not sure if it works we haven't had time to test it, the configs for it were complex and I don't know that we are ready or even really want to support this so I'm removing the references to it in the docs. I will add a comment in the upgrade guide

File: docs/sources/storage/_index.md [hunk 4]
```diff
@@ -51,10 +64,16 @@ S3 is AWS's hosted object store. It is a good candidate for a managed object sto
...
-You may use any subsitutable services, such as those that implement the S3 API like [MinIO](https://min.io/).
+You may use any substitutable services, such as those that implement the S3 API like [MinIO](https://min.io/).
 
 ## Implementations - Index
 
+### Single-Store
+
+Also known as "boltdb-shipper" during development (and is still the schema `store` name). The single store configurations for Loki utilize the chunk store for both chunks and the index, requiring just one store to run Loki.
+
+As of 2.0, this is the recommended index storage type, performance should be nearly as good as the dedicated index types while providing a much less expensive and less complicated deployment.
```
Comment (owen-d) [inline docs/sources/storage/_index.md:75]: Do we need to say _nearly as good_? It's been better in many scenarios and worse in some, but generally by negligible margins.

Comment (slim-bean) [reply docs/sources/storage/_index.md]: fixed

File: docs/sources/upgrading/_index.md [hunk 5]
```diff
@@ -0,0 +1,559 @@
...
+
+If you have an environment to test this in please do so before upgrading against critical data.
+
+There are 2 significant changes warranting the backup of this data because they will make rolling back impossible:
+* A compactor is included which will take existing index files and compact them to one per day and remove non compacted files
+* All index files are now gzipped before uploading
+
+The second part is important because 1.6.0 does not understand how to read the gzipped files, so any new files uploaded or any files compacted become unreadable to 1.6.0 or ealier.
+
+_THIS BEING SAID_ we are not expecting problems, our testing so far has not uncovered any problems, but some extra precaution might save data loss!
```
Comment (owen-d) [inline docs/sources/upgrading/_index.md:147]:
```suggestion
_THIS BEING SAID_ we are not expecting problems, our testing so far has not uncovered any problems, but some extra precaution might save data loss in unforeseen circumstances!
```

File: docs/sources/upgrading/_index.md [hunk 6]
```diff
@@ -0,0 +1,559 @@
...
+    - from: 2020-10-24           ③
+      store: boltdb-shipper
+      object_store: filesystem   ④
+      schema: v11
+      index:
+        prefix: index_
+        period: 24h              ⑤
+```
+① Make sure all of these match your current schema config  
+② Make sure this matches your previoius schema version, Helm for example is likely v9
```
Comment (cyriltovena) [inline docs/sources/upgrading/_index.md:201]:
```suggestion
② Make sure this matches your previous schema version, Helm for example is likely v9  
```

**Review by cyriltovena — APPROVED** [review]
LGTM

with one typo still in there.

**Review by owen-d — APPROVED** [review]

**Pull request merged**